python/metric_pipeline_scripts/jobReporter.py

```diff
@@ -0,0 +1,34 @@
+import numpy as np
+
+from lsst.verify import Job, MetricSet
+from lsst.daf.butler import Butler
+class JobReporter:
+    def __init__(self, repository, collection, metrics_package, spec):
+        self.metrics = MetricSet.load_metrics_package(package_name_or_path='verify_metrics',  # Maybe allow an argument
```

Comment:
I didn't understand the comment "# Maybe allow an argument" here

Reply:
Oh, I'm hard coding the package we get the metrics from.  I was just leaving a note that maybe we should allow this to be passed as an argument instead.  I'll make it clearer.